scripts/install_dependencies.sh

```diff
@@ -3,8 +3,11 @@
 # exit on errors
 set -e
 
+# Get ubuntu distribution code name. All STOL APT debian packages are pushed to S3 bucket based on distribution codename.
+. /etc/lsb-release
+
 # add the STOL APT repository
-echo "deb [trusted=yes] http://s3.amazonaws.com/stol-apt-repository develop main" > /etc/apt/sources.list.d/stol-apt-repository.list
+echo "deb [trusted=yes] http://s3.amazonaws.com/stol-apt-repository ${DISTRIB_CODENAME} main" > /etc/apt/sources.list.d/stol-apt-repository.list
```

Comment:
How to look at the list of available ubuntu distributions support of this carma-clock-1

Reply:
Why would that be necessary. It is not necessary for any other package you install via apt?

Reply:
This package is limited and supported for ubuntu OS only? Also if the ubuntu is version 16.06 or 24.04, does this carma-clock-1 still exist? If not, do we have a visible document that tells users that this package is only compatible with certain ubuntu version?

Reply:
I think this is excessive. When we release CARMA Platform or any other library we can document which ubuntu distributions we have built it on (https://github.com/usdot-fhwa-stol/carma-time-lib/pulls?tab=readme-ov-file#install-library) by listing prerequisites for using the application or library but this does not need to be available from the package manager. Additionally it is definitely not in the scope for this fix. When you install other applications via apt they do not list ubuntu compatibility. We have documented on carma-time-lib how to install this package via apt and provide an ubuntu distribution. Anything beyond that, I do not think is our responsibility.